packages/quantic/build-static-resources.js

```diff
@@ -70,7 +70,7 @@ const LIBRARY_CONFIG = {
         dest: `${STATIC_RESOURCES_PATH}/coveoheadless/recommendation/headless.js`,
       },
       {
-        src: resolveLibraryPath('@coveo/headless', '../dist/definitions'),
+        src: resolveLibraryPath('@coveo/headless', '../../dist/definitions'),
```

Comment:
Small path update; because we moved the "main" to dist/cjs and/or dist/esm, there's one more "level" to go down.

Reply:
@erocheleau, pretty much the only thing that impacts Quantic.

Reply:
Isn't that a breaking change?

Anyone that was potentially relying on the fact that the definitions were in `../dist/definitions` will now be essentially pointing to a broken path?

Reply:
Oh I think I misread it? It's not that the definitions have moved is that this resolve thing starts from the main.js and this one has moved?

Reply:
Okay well as long as this script still correctly builds the right static resource :D 
I'm okay with it